README.md

```diff
@@ -74,18 +74,6 @@ For the [BERT NPI paper](https://arxiv.org/abs/1909.02597) follow the instructio
 Post an issue here on GitHub if you have any problems, and create a pull request if you make any improvements (substantial or cosmetic) to the code that you're willing to share.
 
 
-## Contributing
```

Comment:
Should we add a link to CONTRIBUTING.md somewhere, or is the assumption that most contributors will know to look for that file?

Reply:
That was my assumption. But no harm in advertising the guidelines in the README — restoring the README's contributing section w/ link to CONTRIBUTING.md (2374e45).